docs/tutorials/advanced/classification_of_strategies.rst

```diff
@@ -47,7 +47,7 @@ strategies::
     ... }
     >>> strategies = axl.filtered_strategies(filterset)
     >>> len(strategies)
-    84
+    85
```

Comment:
Travis seems to be suggesting this needs to be bumped to 86

Reply:
And travis is correct! Ta.